src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -63,7 +64,7 @@ public String getDisplayName() {
             return null;
         }
 
-        StringJoiner joiner = new StringJoiner(MainTableFormat.COL_DEFINITION_FIELD_SEPARATOR);
+        StringJoiner joiner = new StringJoiner(JabRefPreferences.COL_DEFINITION_FIELD_SEPARATOR);
```

Comment:
Another comment: the method getDisplayName is called quite often. If we would query a preference for each field in each row having 10k rows, this would lead to performance issues.